### ISSUE TITLE
Name does not exist in current context

### ISSUE THREAD

**Issue by chadamski** [bingcheng45/VitruviusGestureExample#1]
Hey there, 

Trying to work through this example you've provided, but the build is putting out the following errors. Can you help me? I'm a total beginner: 

Build started ...
C:\Sites\kinect-app\get-angles\get-angles\Views\MainWindow.xaml.cs(20,13): error CS0103: The name 'InitializeComponent' does not exist in the current context 
C:\Sites\kinect-app\get-angles\get-angles\Views\MainWindow.xaml.cs(74,21): error CS0103: The name 'lbDisplay' does not exist in the current context 
C:\Sites\kinect-app\get-angles\get-angles\Views\MainWindow.xaml.cs(77,21): error CS0103: The name 'lbDisplay' does not exist in the current context 
C:\Sites\kinect-app\get-angles\get-angles\Views\MainWindow.xaml.cs(80,21): error CS0103: The name 'lbDisplay' does not exist in the current context 
C:\Sites\kinect-app\get-angles\get-angles\Views\MainWindow.xaml.cs(83,21): error CS0103: The name 'lbDisplay' does not exist in the current context

**Comment by bingcheng45** [comment]
Can you show me your MainWindow.xaml.cs code?
just to be sure, are you running it on visual studio 2017?

> 1. 

`C:\Sites\kinect-app\get-angles\get-angles\Views\MainWindow.xaml.cs(20,13): error CS0103: The name 'InitializeComponent' does not exist in the current context` 
This errors seems to be that you do not have `InitializeComponent();` inside your `MainWindow()` method.

> 2.

  `C:\Sites\kinect-app\get-angles\get-angles\Views\MainWindow.xaml.cs(74,21): error CS0103: The name 'lbDisplay' does not exist in the current context` ,
 look in your `MainWindow.xaml` u should create a label in the form and give it a name,  `<Label x:Name="lbDisplay" Content="Label" HorizontalAlignment="Left" Margin="250,147,0,0" VerticalAlignment="Top" FontSize="40"/>`.

do refer to the pdf guide i made found here:  https://bingcheng45.wordpress.com/2017/05/05/step-by-step-kinect-swipe-gesture-tutorial/

it is in STEP #2 - Create a label on a window form title
fig 4

**Comment by chadamski** [comment]
Yeah, I was following your PDF guide, but I also ended up copying the code in this repo just to make sure I wasn't mistyping anything, so the MainWindow.xaml.cs code is [exactly what you have here.](https://github.com/bingcheng45/VitruviusGestureExample/blob/master/VitruviusGestureExample/MainWindow.xaml.cs)

I'll double check again tonight when I get home, but I was having the issue both when I followed your guide to recreate the code and when I simply downloaded your code and used it as-is. And yes, definitely using Visual Studio 2017. Thanks!

**Comment by bingcheng45** [comment]
Yes the code is the same, however you need to create a label in the gui found in the MainWindow.xaml
here is a picture of it: http://imgur.com/a/hpJ6R
drag and drop a label from the toolbar, and give it a name called 'lbDisplay'

**Comment by chadamski** [comment]
Yeah, I did create the label-- I'll double-check to confirm that it's named correctly.

**Comment by bingcheng45** [comment]
is it possible to provide a screenshot of your error or share your project with me? I can take a look into it

**Comment by bingcheng45** [comment]
@chadamski have the issue been solved?
It may be because u renamed the solution to a different name, take a look at this:
https://stackoverflow.com/questions/6925584/the-name-initializecomponent-does-not-exist-in-the-current-context

**Comment by chadamski** [comment]
Hey there, 

Sorry for the delay-- had to get back to my Windows machine.

I'm pretty sure this issue was caused because I was retyping code from your PDF tutorial, and the font you were using in that tutorial made the letter "L" look like the letter "I." I had typed the label as Ib instead of Lb. Doesn't make any sense of course, given the context of Lightbuzz-- I should have known. ;) Perhaps a mono/serif font in the future? Hah. But thanks for your help. I've got new errors now, but I'll need to mess with it for a while to figure out what's going on. Cheers.
